resources/tests/test_resources_queries.py

```diff
@@ -562,3 +571,277 @@ def test_get_harbor_available_active_berths(api_client, pier, available):
     assert executed["data"] == {
         "harbor": {"properties": {"numberOfPlaces": 2, "numberOfFreePlaces": 0}}
     }
+
+
+def test_get_harbor_count(api_client):
+    count = random.randint(1, 10)
+    for _i in range(count):
+        HarborFactory()
+
+    query = """
+        {
+            harbors {
+                count
+                totalCount
+            }
+        }
+    """
+
+    executed = api_client.execute(query)
+    assert executed["data"] == {"harbors": {"count": count, "totalCount": count}}
+
+
+def test_get_harbor_count_filtered(api_client):
+    filtered = random.randint(1, 10)
+    unfiltered = random.randint(1, 10)
```

Comment:
this naming is a bit misleading IMO. `unfiltered` is what you get for `totalCount`, both of these will be `filtered` results, but one `withElectricityInPiers` while the other one is `withoutElectricityInPiers` 🤔 same applies for other test cases too

Reply:
Yeah, I was having the same concern but couldn't come up with a clearer naming. I'll think about it and suggestions are also welcome. 😅

Reply:
Great job with the names!